enhancements/cloud-integration/aws/aws-sts-support.md

```diff
@@ -87,15 +101,26 @@ All CVO managed operators will also now mount in a [Kubernetes Projected Service
 
 openshift-install will be modified to accept user provided service account signing keys and configure the Kubernetes APIServer to use them.
 
-For each `CredentialsRequest` the user will need to manually create the relevant Secret and role
-Users will have to manually provision the IAM roles, policies, and Secrets for each `CredentialsRequest`. This process will also require the name of the `ServiceAccount` that the pod consuming the credentials uses, which is typically created by the operator itself. However, we do not presently have the `ServiceAccount` name modeled in the `CredentialsRequest` API, and would like to defer doing this until we have more real world data and experience before making a permanent API commitment. As such, credentials will be documented as a list policy permissions, service account names, and target secrets, which the user will have to iterate:
+For each `CredentialsRequest` the user will need to manually create
+the relevant Secret and role.  Users will have to manually provision
+the IAM roles, policies, and Secrets for each
+`CredentialsRequest`. This process will also require the name of the
+`ServiceAccount` that the pod consuming the credentials uses, which is
+typically created by the operator itself. However, we do not presently
+have the `ServiceAccount` name modeled in the `CredentialsRequest`
+API, and would like to defer doing this until we have more real world
+data and experience before making a permanent API commitment. As such,
+credentials will be documented as a list policy permissions, service
+account names, and target secrets, which the user will have to
+iterate:
 
   1. Create an IAM Role with no policy attached, type = web identity, identity provider created by the admin manually.
     1. If the role will not be shared across multiple clusters and should be removed when the cluster is deprovisioned, tag it with `kubernetes.io/cluster/[infraid]=owned`.
     1. Note that openshift-install destroy cluster will not cleanup the OIDC provider itself today.
   1. Attach an in-line policy to the role which matches the permissions in the `CredentialsRequest` or documentation.
   1. Ensure the trust relationship for the Role points to the cluster’s AWS IAM OIDC ARN with conditions to limit which `ServiceAccount` can assume this role.
-     ```json
+
+```json
```

Comment:
this was probably intended to be indented to nest under the `Ensure the trust relationship...` point.